src/main/java/com/squareup/kotlinpoet/PropertySpec.kt

```diff
@@ -98,6 +100,15 @@ class PropertySpec private constructor(builder: Builder) {
     }
   }
 
+  fun fromPrimaryConstructorParameter(parameter: ParameterSpec): PropertySpec {
```

Comment:
Not 100% sure we want this in API, probably better to go with `internal` for now